app-flow-router.html

```diff
@@ -103,10 +103,17 @@
                     throw new TypeError('Attribute config for app-flow-router used in "' + this.dataHost.is + '" is not set ');
                 } else {
                     var conf = this._configObject[this.current + flowEvent.event];
+
+                    if (!conf && flowEvent.event === 'menuitem-tapped') {
```

Comment:
I think i got your intention. But cant you do it with a special target config like HISTORY-BACK?

 if (conf.target === 'DYNAMIC-TARGET') {
...

Reply:
sure, i will analyze it tomorrow and update the pr.
Thx for the tip.

Reply:
Thank you for the idea and the pull request.

Reply:
I guess you want to specifiy a dynamic current as fallback. I mean, you have different menu items and you can trigger this items from a dynamic current view. So, you can do it like this example ['*', 'settings-tapped', '/settings'] Now, you can reach the setting page from every location.

Reply:
The problem is a little bit different: it's not about the fallback route. With the current implementation you can't define multi-routes for one event name, but we need that in our application for navigation from the left-drawer-menu. You can open the menu on every page, so you will have to have the possibility to define a generic route for it (which means multi-sources for one event name)

The example that you given is a nice idea how to solve the problem, but it won't work right now. The current implementation of app-flow router does not support wildcards. There is a line in code:
var conf = this._configObject[this.current + flowEvent.event];
When this runs against your example (['*', 'settings-tapped', '/settings']) it will return undefined (this.current will never equal '*', but the real view name). When conf is undefined the router prints warning and it ignore saving route in the history.

Reply:
multiple-sources for one event name is implemented in 1.4.1
Thank you for the input.